source/accounts/index.rst

```diff
@@ -4,10 +4,25 @@
 Accounts
 ########
 
-.. note::
+Getting an Internal RDHPCS Account
+==================================
+
+What NOAA Does
+---------------
+
+#. Conduct security investigation (6-12 weeks).  This includes a fingerprint check, background check, and DOC public trust investigation
+#. Provide a **@noaa.gov** email account and initial password. You may have access to the NOAA email account before the security review is complete. Once you have a noaa.gov email address, you also have access to several internal systems.
 
-   System maintenance will affect access to RDHPCS systems. Click `here <https://calendar.google.com/calendar/u/1/r?cid=bm9hYS5nb3ZfZjFnZ3U0M3RtOWxmZWVnNDV0NTlhMDYzY3NAZ3JvdXAuY2FsZW5kYXIuZ29vZ2xlLmNvbQ>`_ to view the RDHPCS Maintenance Downtime Calendar.
+What You Do
+-----------
 
+#. Take the NOAA IT Security Awareness Training (annual requirement).
+#. Use the Account Identity Management (AIM) system to request access to a Vetting Project. You access AIM using your NOAA.gov email address and password.
```

Comment:
Nitpicky -- should "vetting" be capitalized or not?  (Should be consistent either way...)